packages/edit-site/src/components/dataviews/item-actions.js

```diff
@@ -2,18 +2,28 @@
  * WordPress dependencies
  */
 import {
-	DropdownMenu,
-	MenuGroup,
-	MenuItem,
 	Button,
 	Modal,
 	__experimentalHStack as HStack,
+	privateApis as componentsPrivateApis,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
-import { useMemo, useState } from '@wordpress/element';
-import { moreVertical } from '@wordpress/icons';
+import { useMemo, useState, Fragment, Children } from '@wordpress/element';
+import { moreVertical, Icon } from '@wordpress/icons';
 
-function PrimaryActionTrigger( { action, onClick } ) {
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+
+const {
+	DropdownMenuV2: DropdownMenu,
```

Comment:
Would it be possible to use `DropdownMenuV2Ariakit` & related subcomponents instead? 

`DropdownMenuV2` (the radix-based version) is not maintained anymore. I'm in the process of migrating all existing usages to the ariakit-based version, and I'll remove this component completely once the migration is over. (components naming will also improve after that!)

There are minor differences in terms of APIs, you can take a look at the [Storybook file](https://github.com/WordPress/gutenberg/blob/fc5619d6ac11605683f4a72dcea53e92d20f4410/packages/components/src/dropdown-menu-v2-ariakit/stories/index.story.tsx) to see a few examples.

Reply:
I'll give it a go again. I tried at first and had some issues, that I'll remember if I try again 😆 . I think it had to do with `align` and placement and something with the trigger I think. I'll post soon with the exact issues.

Reply:
In the new component, you should use the `placement` prop (same as our `Popover`) instead of the `align` prop, but let me know in case there's more. Also happy to pair on it quickly if that helps.

Getting feedback from folks like you when using the component is actually very useful :)

Reply:
@ciampo I pushed an update to use the new component, but I can't make it look like this(alignment)
<img width="344" alt="Screenshot 2023-11-24 at 3 47 33 PM" src="https://github.com/WordPress/gutenberg/assets/16275880/5351236d-b38e-48eb-9d08-635a0a02ce30">

Reply:
> while the placement seems to work well, in story book it doesn't

Could you be more specific?

Also, there may be two factors that affect testing `placement` in Storybook:
- a `floating-ui` bug which causes all popover-based components to have an extra offset when the left sidebar is opened. This bug can be avoided by closing the left sidebar when testing the Storybook examples.
- The fact that the dropdown trigger (button) is rendered in the top left of the viewport means that a lot of placements are going to naturally "flip" because there isn't enough space. I suggest adding some margin/padding around the button to test such placements

Let me know if you're still facing issues, though!